### DISCUSSION TITLE
Support the new/upcoming generic ZHA Zeroconf discovery?

### DISCUSSION THREAD

**Discussion by Hedda** [tube0013/tube_gateways#206] [Ideas]
puddly is working on a new new/upcoming generic Zeroconf discovery for the ZHA integration in Home Assistant to allow it to discover new network-attached Zigbee Coordinator adapters without code changes to the ZHA component in Home Assistant core. Please look into adding support for this Zeroconf discovery formatting, see:

https://github.com/home-assistant/core/pull/126294

The goal with this is that code changes to the ZHA component in Home Assistant core but it puts some new requirement on the firmware site where the manufacturer need to enter unique metadate for each new type and model of network-attached Zigbee Coordinator adapter. This will eventeully also make the old disovery method obeslete:

https://www.home-assistant.io/integrations/zha#discovery-via-usb-or-zeroconf

PS: Alternative is to use the old method but it is likley that new patches to Home Assistant core will be accepted once support for this new formallting/method has been implemented.

**Comment by tube0013** [comment]
Yes I was somewhat involved with this.
